easybuild/easyblocks/c/cuda.py

```diff
@@ -89,8 +89,7 @@ def sanity_check_step(self):
         custom_paths = {
             'files': ["bin/%s" % x for x in ["fatbinary", "nvcc", "nvlink", "ptxas"]] +
                      ["%s/lib%s.so" % (x, y) for x in chk_libdir for y in ["cublas", "cudart", "cufft",
-                                                                           "curand", "cusparse"]] +
-                     ["open64/bin/nvopencc"],
```

Comment:
so, this is no longer around for recent versions? we should only drop this for recent versions then?

something like this:

```python
extra_files = []
if LooseVersion(self.version) < LooseVersion('7.5'):
    extra_files.append('open64/bin/ncopencc')

custom_paths = {
    ...
```

and then use `extra_files` in here?